websocket/websocket-enums.go

```diff
@@ -0,0 +1,11 @@
+package websocket
+
+//Defines Websocket enums
+
+const RequestTypeChatRoom = "connect_to_chat_room"
+const RequestTypeSendMessage = "send_message"
+const RequestParamRoomId = "room_id"
+const RequestParamSenderId = "sender_id"
+const RequestParamMessageFrom = "message_from"
+const RequestParamMessage = "message"
+const RequestParamRequestType = "requestType"
```

Comment:
```suggestion
const (
 RequestTypeChatRoom = "connect_to_chat_room"
 RequestTypeSendMessage = "send_message"
RequestParamRoomId = "room_id"
RequestParamSenderId = "sender_id"
RequestParamMessageFrom = "message_from"
RequestParamMessage = "message"
RequestParamRequestType = "requestType"
)
```
Arrange them in alphabetical order

Reply:
Done